src/app/service-providers/components/InpiFilter.tsx

```diff
@@ -0,0 +1,30 @@
+import { Fieldset } from '@headlessui/react'
+
+import { CheckboxesContainer } from './CheckboxesContainer'
+import { CheckboxWithLabel } from './CheckboxWithLabel'
+import { FilterHeading } from './FilterHeading'
+import { useFilterQueryState } from '../hooks/useFilterQueryState'
+
+type InpiFilterProps = {
+  options: Array<string>
+}
+
+export function InpiFilter({ options }: InpiFilterProps) {
```

Comment:
Same for filename, etc?

```suggestion
export function IpniFilter({ options }: IpniFilterProps) {
```